File: projetoES3/src/dao/IDao.java
package dao;

import java.util.List;
import model.domain.EntidadeDominio;

public interface IDao {

	public void salvar(EntidadeDominio entidade);

	public void alterar(EntidadeDominio entidade);

	public void inativar(EntidadeDominio entidade);
	
	public List<EntidadeDominio> consultar(EntidadeDominio entidade);
	
}

File: typeScript/src/Dao/Idao.ts
import {EntidadeDominio} from "../model/domain/EntidadeDominio";

export interface IDao {

       salvar( entidade: EntidadeDominio):  void

      alterar(entidade: EntidadeDominio): void

      inativar(entidade: EntidadeDominio): void

      consultar(entidade: EntidadeDominio|undefined): any

}